### ISSUE TITLE
how to install voxce?

### ISSUE THREAD

**Issue by khpavel** [trialmanager/voxce#1]

**Comment by khpavel** [comment]
is any instruction to install program  locally?

**Comment by benchoufi** [comment]
yes, I am really not familiar to servlet apps. And the architecture seem not to comply tomcat server deployment rules

**Comment by lovish1988** [comment]
It's a eclipse project.

**Comment by palazzoantonio** [comment]
How Install ?
1. prepare server tomcat - ok 
2. copy content of folder "WebContent" in webapp folder tomcat . ( I see app in Application manager of tomcat web manager) 
3. then ?

**Comment by trialmanager** [comment]
Please see the wiki page https://github.com/trialmanager/voxce/wiki

**Comment by palazzoantonio** [comment]
Sorry , but the script SQL to generate database has error , but i resolve it and now database structure is ready ,
BUT  java application not start . There are file to edit before start application ? 
where is the file with database configurazione information (user/password to connect db) ?
Can you help me ?

**Comment by trialmanager** [comment]
@lovish1988 can you please help?

**Comment by lovish1988** [comment]
First u have to prepare war file of this project. Deploy war file using tomcat manager ui

**Comment by rober7o** [comment]
I found errors in the sql file, can somebody helpme with the sql file :(

**Comment by rober7o** [comment]
@palazzoantonio how you fix the database, because some inserts are uncomplete and it generate exceptions @trialmanager @lovish1988
